Project.toml

```diff
@@ -5,16 +5,20 @@ version = "0.2.1"
 
 [deps]
 Artifacts = "56f22d72-fd6d-98f1-02f0-08ddc0907c33"
+AutoHashEquals = "15f4f7f2-30c1-5605-9d31-71845cf9641f"
 DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
 Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
 DelimitedFiles = "8bb1440f-4735-579b-a4ab-409b98df4dab"
 Distributed = "8ba89e20-285c-5b6f-9357-94700520ee1b"
 EarthOrientation = "732a3c5d-d6c0-58bc-adb1-1b51709a25e2"
 HORIZONS = "5a3ac768-beb4-554a-9c98-3342fe3377f5"
+HTTP = "cd3eb016-35fb-5094-929b-558a96fad6f3"
 Healpix = "9f4e344d-96bc-545a-84a3-ae6b9e1b672b"
 InteractiveUtils = "b77e0a4c-d291-57a0-90e8-8db25a27a240"
 IntervalArithmetic = "d1acc4aa-44c8-5952-acd4-ba5d80a2a253"
+IntervalRootFinding = "d2bf35a9-74e0-55ec-b149-d360ff49b807"
 JLD = "4138dd39-2aa7-5051-a626-17a0bb65d9c8"
+JLD2 = "033835bb-8acc-5ee8-8aae-3f567f8a3819"
```

Comment:
Currently, we cannot remove `JLD` since all artifacts (except for `PlanetaryEphemeris` results) are saved in `.jld` files.

Reply:
Ah, you're right! I then suggest to migrate the rest of the artifacts from JLD to JLD2 independently (I'll open an issue); just to avoid mixing that matter with the current scope of your PR